.github/workflows/pr-title.yaml

```diff
@@ -7,8 +7,8 @@ on:
       - edited
       - synchronize
 
-permissions:
-  pull-requests: read
+# permissions:
```

Comment:
Just disabling this entirely for now - should resolve the PR title check issues (I hope)